docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: "3.7"
+services:
+    wildduck:
+        network_mode: host
```

Comment:
Don't use host as network mode. You'll end up exposing any host communication on the container, potentially exposing secret data.

Reply:
Mostly put the docker compose as a sample file. I could also, remove network mode host and pass in the the mongo/redis url as command line args to wildduck, as an alternative